libs/model/src/community/GetMembers.query.ts

```diff
@@ -61,6 +65,18 @@ export function GetMembers(): Query<typeof schemas.GetCommunityMembers> {
           case 'not-in-group':
             membershipsWhere = `AND NOT EXISTS (${membershipsWhere} AND "Memberships".reject_reason IS NULL)`;
             break;
+          case 'allowlisted':
+            membershipsWhere =
+              addressBinding && addressBinding.length > 0
+                ? `AND "Addresses".address IN(:addressBinding)`
+                : '';
+            break;
+          case 'not-allowlisted':
+            membershipsWhere =
+              addressBinding && addressBinding.length > 0
+                ? `AND "Addresses".address NOT IN(:addressBinding)`
+                : '';
```

Comment:
With the new approach, it shouldn't be labeled as `allowlisted`/`not-allowlisted` as the filter since the filter is no longer based on the existing `allowList` logic in the Group model. Please use another name to avoid confusion.

I think this is great because it's a more general-use filter that can be used beyond allowlist, but needs a new name.

Reply:
🙏